ports/libarchive/portfile.cmake

```
@@ -1,25 +1,30 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO libarchive/libarchive
-    REF 1b2c437b99b361c7692538fa373e99955e9b93ae      #v3.5.2
-    SHA512 df527dd333b01ed85f07831ba0bd4b1d0b5384fe12cfa53474ad39c04509105a3c8574a2d21a430e3584a931c8f6ae923bca95df83945f0c593c1ffaed3f62da
+    REF 6c3301111caa75c76e1b2acb1afb2d71341932ef      #v3.6.1
+    SHA512 2fd56ac20e4249807174a2ae29de1cbca55c8f8f247500845f56fd1fd9ebf48c17b8a25a93156df71df9526c0061415ec7d72a6b46bbaca776047e381a2321a7
     HEAD_REF master
     PATCHES
         disable-warnings.patch
         fix-buildsystem.patch
         fix-cpu-set.patch
-        fix-dependencies.patch
+        fix-deps.patch
         pkgconfig-modules.patch
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     FEATURES
         bzip2   ENABLE_BZip2
+        bzip2   CMAKE_REQUIRE_FIND_PACKAGE_BZip2

```

```suggestion
        bzip2   ENABLE_BZip2
        bzip2   CMAKE_REQUIRE_FIND_PACKAGE_BZip2

```
@Neumann-A  Does the same feature require two options name? 
Why use the function `CMAKE_REQUIRE_FIND_PACKAGE`? 
If the upstream dependency on these ports is changed to `find_package`, will it break the logic of the upstream code?